.goreleaser.yml

```diff
@@ -2,6 +2,10 @@
 version: 2
 project_name: govmomi
 
+env:
+  - GO_VERSION=1.23.0
```

Comment:
Does `1.23.0` pin to that patch release? Or would using `1.23` instead pickup the latest patch release (1.23.8) ?
We can also use latest 1.24 for release bits, just need to use 1.23 for the unit-test matrix to make sure we still work with that version.

Reply:
This is using the specific tag. Probably best to use 1.24 as the base image.

Reply:
Updated to `golang:1.24` in [a5f933d](https://github.com/vmware/govmomi/commit/a5f933dcb97624c3c783489f58a68b84d0691203).